VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -24,7 +22,7 @@
     private String end; // completion date
     private String start_display;
     private StartType start_type;
-    private String name;
+    @NonNull private String name;
```

Comment:
Hmm, would be nice to have nullability declared for all the fields here, but I suppose it's out of the scope of this pull request. Maybe we should create a ticket for this purpose. (We should also have a ticket for actually validating these non-null requirements I guess.)